src/libraries/Microsoft.Extensions.Primitives/src/Microsoft.Extensions.Primitives.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);netcoreapp3.0;netstandard2.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);CS1591;CA1200;SA1121;SA1028;SA1129;SA1027</NoWarn>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <ExcludeCurrentNetCoreApp>true</ExcludeCurrentNetCoreApp>
```

Comment:
Is this for the package?

I thought the property was: 

`<ExcludeFromPackage Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">true</ExcludeFromPackage>`

Reply:
This is a shortcut if you don't want to type the whole thing.

Reply:
I see. Should we prepend a `FromPackage` suffix? To me reading that, means: don't include this assembly in current netcoreapp framework.

Reply:
I agree it's confusing out of context.  I would say `ExcludeNetCoreAppCurrentFromPackage` to match the property name.  At the end of the day it might not be saving that much in total char count over just the shorter prop + condition.

Reply:
@Anipik what are your thoughts?

Reply:
Sure we can change it to ```ExcludeNetCoreAppCurrentFromPackage ```